packages/widgetbook_exception/lib/src/widgetbook_exception.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022, Jens Horstmann
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
```

Comment:
Have you added this? In my opinion, it's unnecessary but it might have been generated by the mason brick. I think it's a bit redundant to have this because if we start with adding a file header now, we have to be consistent with it and I'm not seeing it because it adds unnecessary work 😀